lib/opensusebasetest.pm

```diff
@@ -858,6 +859,16 @@ sub grub_select {
     elsif (!get_var('S390_ZKVM')) {
         # confirm default choice
         send_key 'ret';
+        if (get_var('USE_SUPPORT_SERVER') && is_aarch64 && is_opensuse)
+        {
+            # On remote installations of openSUSE distris on aarch64, first key
+            # press doesn't always reach the SUT, so introducing the workaround
+            wait_still_screen;
+            if (check_screen('grub2')) {
+                record_info 'return key did not reach the system, re-trying';
```

Comment:
I would suggest to use two params, for example, 're-try' and the long message. I don't remember, but I guess with only one it shows empty box and then the message is visible when you click, still fine, just an enhancement.

Reply:
We have this test on SLES and on 64bit TW. As for `record_info` that's good suggestion, will fix that. Thanks!